### ISSUE TITLE
Trocar mural de pessoas aleatorias por fotos de membros do grupo 

### ISSUE THREAD

**Issue by cefigueiredo** [rubyonrio/rubyonrio.github.com#19]
Galera,

Já há algum tempo, nós começamos a discutir a troca do mural de fotos, que hoje só tem fotos de pessoas aleatórias, por um mural com as fotos dos membros do grupo.

Visto que já existe a lista com os hashs, pretendo começar a trabalhar nela.

**Comment by GiovanniSM20** [comment]
Ainda faz sentido tocar essa issue ?

**Comment by tauil** [comment]
Se estiver interessado, sim! Depois de um tempo eu pensei que dá pra fazer de forma muito mais "simples" do eu no projeto que listei. Basta listar os contributors e montar a imagem com css...

**Comment by rafa-so** [comment]
Boa tarde a todos, aqui ainda faz sentido essa feature?

**Comment by giovannism20** [comment]
> Boa tarde a todos, aqui ainda faz sentido essa feature?

Acredito que sim

**Comment by tauil** [comment]
Sim! Essa é uma tarefa relativamente fácil e boa para quem está começando...

**Comment by vagnerzampieri** [comment]
A issue mais longa da história da humanidade, demorou só quase 12 anos para ser feita hahahahaha

https://github.com/rubyonrio/rubyonrio.github.com/commit/57b64b8c8ceaaaefc939d8f654491ffa627de737
https://github.com/rubyonrio/rubyonrio.github.com/commit/2323db640f2c17edbd09b17563a6e0741bc21770
